software/controller/lib/core/comms.cpp

```diff
@@ -134,6 +134,10 @@ static void process_rx(GuiStatus *gui_status) {
   }
 }
 
+Time comms_get_lastrx_time(void) {
+  return last_rx;
```

Comment:
This is returning a variable that was initialized as global variable in `alarms.cpp`? This is extremely confusing.

Reply:
not needed, instead changed the code and using a comms_is_time_to_raise_commfail_alarm() directly form comms.cpp